### PR TITLE
Add Rivalry ability

### PR DESCRIPTION
**Pull request by Ntonio36**

### CODE REVIEW

**Comment by Zarel** [comment]
Did you just add a gender selector to both sides?

Honestly, the main thing that needs to be fixed in the damagecalc is UX.

**Comment by Ntonio36** [comment]
I reckon I wasn't quite inspired on the fact of adding (yet) a(nother) selector, but there was probably no other way around... unless there are some tricks or other stuff I don't know about yet.

File: js/data/pokedex.js [hunk 1]
```diff
@@ -2110,8 +2110,6 @@ var POKEDEX_RBY = {
 };
 
 var POKEDEX_GSC = $.extend(true, {}, POKEDEX_RBY, {
-	"Magnemite": {"t2": "Steel"},
```
Comment (AustinXII) [inline js/data/pokedex.js]: I like having gen changes at the top of the array for convenience

**Comment by AustinXII** [comment]
@Zarel 100% true, maybe someone with more free time is willing to update the calc design, perhaps ask some of the “new” devs. A revamp in general would be superb, the old look was fine for 2014, but extremely boring for 2018

**Pull request merged**